src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/Footer.tsx

```diff
@@ -31,11 +32,12 @@ export const Footer = () => {
   const translate = useTranslate()
   const dispatch = useAppDispatch()
   const swapperName = useAppSelector(selectActiveSwapperName)
+  const activeQuote = useAppSelector(selectActiveQuote)
   const lastHopBuyAsset = useAppSelector(selectLastHopBuyAsset)
   const tradeExecutionState = useAppSelector(selectTradeExecutionState)
   const networkFeeUserCurrency = useAppSelector(selectTotalNetworkFeeUserCurrencyPrecision)
   const sellAmountBeforeFeesUserCurrency = useAppSelector(selectQuoteSellAmountUserCurrency)
-  const { isModeratePriceImpact } = usePriceImpact()
+  const { isModeratePriceImpact } = usePriceImpact(activeQuote)
```

Comment:
Since `usePriceImpact` is a hook itself, any reason why it doesn't select the `activeQuote` itself and has it passed down instead?

Reply:
Reading the implementation of `usePriceImpact`, this now makes sense 🧠